bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableIO.java

```diff
@@ -760,6 +754,65 @@ public void finishBundle(Context context) throws Exception {
   }
 
   /**
+   * A {@link DoFn} that can write either a bounded or unbounded {@link PCollection} of
+   * {@link Mutation}s to a table specified via a {@link CloudBigtableTableConfiguration}.
+   */
+  public static class CloudBigtableSingleTableWriteFn
```

Comment:
This should not be the default implementation.  Buffered mutator should be the default.